app/ModelFunctions/PhotoFunctions.php

```diff
@@ -140,6 +140,54 @@ public function createThumb(Photo $photo, string $frame_tmp = '')
 		return true;
 	}
 
+	/**
+	 * Returns 'photo' if it is a photo
+	 * Returns 'video' if it is a video
+	 * Returns 'raw' if it is an accepted file (we only check extensions)
+	 * Returns 'error message' if it is something else.
+	 *
+	 * @param $file
+	 * @param $extension
+	 *
+	 * @return string
+	 */
+	private function kind($file, string $extension)
```

Comment:
Is `kind` a good function name here when we're using `kind` for big/medium/small/etc in similar code paths?